src/server/zset_family.cc

```diff
@@ -2682,6 +2740,7 @@ void ZSetFamily::Register(CommandRegistry* registry) {
       << CI{"ZCARD", CO::FAST | CO::READONLY, 2, 1, 1, acl::kZCard}.HFUNC(ZCard)
       << CI{"ZCOUNT", CO::FAST | CO::READONLY, 4, 1, 1, acl::kZCount}.HFUNC(ZCount)
       << CI{"ZDIFF", CO::READONLY | CO::VARIADIC_KEYS, -3, 2, 2, acl::kZDiff}.HFUNC(ZDiff)
+      << CI{"ZDIFFSTORE", kStoreMask, -4, 3, 3, acl::kZDiffStore}.HFUNC(ZDiffStore)
```

Comment:
the minimal number of args to zdiffstore is 3 f.e
zdiffstore out 1 zset
is a valid command

Reply:
I saw your email, you are correct.
I will check later again our documentation here is a bit confusing.
You can ignore this comment